plugins/mumble_plugin_win32.h

```diff
@@ -18,9 +18,24 @@
 
 #include "mumble_plugin.h"
 
+#if _MSC_VER < 1800 && defined(M_IX86)
+typedef unsigned long long uintptr64_t;
+typedef unsigned long uintptr32_t;
+#else
+#include <stdint.h>
+typedef uint64_t uintptr64_t;
```

Comment:
I picked the names because I felt that "uintptr32_t" and "uintptr64_t" would be immediately familiar to anyone who've used "uintptr_t" before.

I don't really like moduleptr32_t because... it isn't a module pointer, it's just a pointer.

Maybe ptr32_t? PTR32? POINTER32? I dunno. I still quite like uintptr32_t.